### DISCUSSION TITLE
Show accuracy while typing

### DISCUSSION THREAD

**Discussion by frogfrogfrogfrogfrog** [monkeytypegame/monkeytype#676] [Ideas] [answered]
I would love to be able to see my accuracy while I type. Also being able to toggle decimals on and off with the accuracy.

**Comment by Miodec** [comment, marked as answer]
Im not exactly sure if this is useful information, especially while trying to type fast.

**Reply by frogfrogfrogfrogfrog** [reply]
I get your point that people aren't going to need to see their accuracy when trying to type fast. I really only need this for two purposes. The first being that since min accuracy exists now (thank you for adding that in) I'd like to know how close I am to my accuracy limit and the second use being for longer tests and I'd like to see how my accuracy fluctuates as I type instead of just seeing the accuracy at the end. 

If those two reasons aren't enough for you to add them in that's fine.

**Reply by Miodec** [reply]
The other problem is where would i place it. With large text above and below is already taken up by live wpm and progress.

**Reply by frogfrogfrogfrogfrog** [reply]
I was thinking that you have the wpm and accuracy both on the bottom with wpm on the left side and accuracy on the right 

if accuracy or wpm are by themselves you could then just put them in the center

**Reply by Miodec** [reply]
Alright, sure. Will be added soon
